src/main/java/frc/robot/InitBindings.java

```diff
@@ -58,7 +59,22 @@ public static void initDriveBindings(Drive drive) {
                 },
                 drive));
 
-    // pov right (reef 0-11 -> processor left -> processor right )
+    leftJoystick
+        .button(1)
+        .onTrue(
+            new InstantCommand(
+                () -> {
+                  drive.lockRotationToPosition(new Translation2d(3.5, 3));
+                },
+                drive));
+    leftJoystick
```

Comment:
just for future reference, you can add the `.onFalse()` after the `.onTrue()` rather than re-referencing the button.